README.md

```diff
@@ -35,6 +35,11 @@ Activate the module by adding it to the config.js file as shown in the example b
 | items            | The rows of data you want the module to show. See list below. By default will show all available            | `['battery','range-estimated','locked','odometer']` |
 | initialLoadDelay | How many seconds to delay initial API call                                                                  |
 | dataTimeout      | How old data must be in seconds before 'data-time' is displayed. Use 0 to always show                       | `0`                                                 |
+| googleMapApiKey  | Google Maps "Static Maps API" | `AIzaSyB6KgHKwRNa63JsVHuu7d8jV-1IH875idKs` |
```

Comment:
Just out of interest, is that a valid key? Wouldn't be a good idea to publish one of your personal keys on here!